ere-guests/stateless-validator/reth/pico/Cargo.toml

```diff
@@ -17,10 +18,7 @@ reth-primitives-traits = { workspace = true, features = [
     "serde-bincode-compat",
 ] }
 
-revm = { workspace = true, default-features = false, features = [
-    "kzg-rs",
```

Comment:
The new `revm` version has the `kzg-rs` removed since now the no_std way is using arkworks.

Reply:
@kevaundray, I ended up doing hte CryptoProvier strategy for Pico. If [you're curious](https://github.com/eth-act/zkevm-benchmark-workload/pull/205/commits/c956b890fc77d6dca4e16a2a0dbdae32fad5508b).

cc @han0110 (since you might be interested too).